source/api/presentation/3.0/change-log.md

```diff
@@ -1,125 +1,263 @@
-major: 2
-minor: 1
-patch: 1
+major: 3
+minor: 0
+patch: 0
 pre: final
 redirect_from:
-  - /api/presentation/2.1/change-log-211.html
+  - /api/presentation/3.0/change-log-30.html
 ---
 
-This document is a companion to the [IIIF Presentation API Specification, Version 2.1.1][prezi-api]. It describes the editorial changes to the API specification made in this patch release, such as clarifications and typo corrections. It also describes corrections to related documents that are not [semantically versioned][semver], such as example resources and resources to manage transformation to and from the JSON-LD serialization.
-
-For the significant changes between 2.0 and 2.1, please see the [2.1 Change Log][changelog-21].
+This document is a companion to the [IIIF Presentation API Specification, Version 3.0][prezi30]. It describes the changes to the API specification made in this major release, including ones that are backwards incompatible with [version 2.1.1][prezi21], the previous version.
 
 ## Table of Contents
 {:.no_toc}
 
 * Goes Here
 {:toc}
 
-## 1. Editorial Changes
 
-### 1.1. Clarify use of `viewingHint`
+## 1. Breaking Changes
+
+### 1.1. External Specifications
+
+#### 1.1.1. Use JSON-LD 1.1
+
+JSON-LD remains the core serialization of the Presentation API. For version 3.0, some features of the JSON-LD Community Group specification make a significant improvement to the API's structure and consistency. While this specification is not a W3C Technical Recommendation at the time of release, the likelihood of the standardization process being successful is extremely high and the rewards have been judged to be worth the minimal risk of unintended incompatibility. Scoped contexts, language sets, and the ability to use `@none` in language maps make implementation significantly easier. See issue [#1192](https://github.com/IIIF/api/issues/1192).
+
+#### 1.1.2. Use Web Annotation Data Model
+
+Annotations remain a core feature of the Presentation API. Instead of the community specified Open Annotation model, version 3.0 adopts the W3C standard Web Annotation Data Model. The practical effects are limited but real, including changes to property names and classes. See issue [#496](https://github.com/IIIF/api/issues/496).
+
+
+### 1.2. Property Naming and Semantics Changes
+
+Several existing properties were renamed for consistency, developer convenience, or to better reflect the intended semantics. Some of the semantics were also clarified based on implementation experience during previous versions.
+
+#### 1.2.1. Rename `@id` to `id`, `@type` to `type`
+
+These properties were renamed to enable javascript developers to use the "dot notation" (`manifest.id`) instead of the square brackets based equivalent needed with the `@` character (`manifest['@id']`). This follows JSON-LD community best practices established by schema.org, the JSON-LD, Web Annotation and Social Web working groups. See issue [#590](https://github.com/IIIF/api/issues/590).
```

Comment:
Javascript (or JavaScript)
square-brackets-based (or something similar)

Reply:
I think the stronger association is square with brackets, so "square-brackets based"?

Reply:
It would be "brackets-based equivalent" w/o "square" so I think "square-brackets-based equivalent" may be correct.  The APA style guide has "non-achievement-oriented students", which seem parallel.   Not 100% certain though.

Reply:
:+1: